darwin/package.json

```diff
@@ -13,25 +13,27 @@
     "electron": "electron .",
     "player": "./bin/player-window.sh",
     "production": "NODE_ENV=production electron .",
+    "lint": "cd .. && npm run lint darwin && cd darwin",
```

Comment:
I think that eslint should only run from the root package. there is no difference in the os, it's all the same js with the same rules. what's the reason/advantage in doing it this way?

Reply:
I agree, I just thought that having a shortcut would be useful since we usually work on a single OS at a time. The shortcut is literally just going to the root to run lint. I can remove it if you prefer, so people don't get confused.

Reply:
I prefer it in the root, yes. I found it quite confusing already 😄